### ISSUE TITLE
Not booting after upgrade

### ISSUE THREAD

**Issue by cpuguy83** [docker/for-mac#657]
### Expected behavior
Should boot
### Actual behavior
😢 
### Information

  - Diagnostic ID: F9467209-AD82-445C-88DC-A760AED26E30
  - OSX 10.11.6

### Steps to reproduce the behavior

  1. Upgrade to beta26

**Comment by cpuguy83** [comment]
Found the issue....
I have `daemon.json` set in the config database but it's empty in the VM.
It happens to be that at one point I was using `overlay2` so there is a graph dir for both overlay2 and aufs.
Since the daemon.json is now empty in the VM (still set in OSX) docker is bailing out:
```
time="2016-09-17T11:57:03.942185295Z" level=fatal msg="Error starting daemon: error initializing graphdriver: \"/var/lib/docker\" contains several valid graphdrivers: aufs, overlay2; Please cleanup or explicitly choose storage driver (-s <DRIVER>)"
```

In addition I noticed that my CPU's went from being set to 8 to being at 4, and my RAM from 4GB to 2GB.

**Comment by justincormack** [comment]
Yes that has happened to me too. You don't really have it set - do a git `reset --hard` and then set it again. The database defaults to a bare git repo as it doesnt update the checked out copy.

You really should do a factory reset when changing graph drivers. I have some code to try to work out what to do but i dont think I merged it yet as we are still not supporting overlay2 officially (although I use it mostly). I think the unmerged PR forces aufs if an aufs directory exists and you dont specify a storage driver in the database. Maybe I should merge this part anyway (it used to specify aufs as default in the database, but I changed that to nothing).
